HW_1/task_1.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+print('Введите трехзначное число')
+a = int(input())
```

Comment:
Функцию `print` использовать необязательно. Вывод сообщения перед вводом можно осуществлять через функцию `input`.
```suggestion
a = int(input('Введите трехзначное число'))
```